packages/synapse-react-client/src/components/SynapseHomepage.tsx

```diff
@@ -363,67 +434,271 @@ export const SynapseHomepage: React.FunctionComponent<SynapseHomepageProps> = ({
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails sx={{ padding: 0 }}>
+              <SynapsePriceFeatureRow
+                featureDescription="Option to make data available through controlled-access"
+                managedPlanValue={true}
+                dataCoordinationPlanValue={true}
+              />
+            </AccordionDetails>
+          </Accordion>
+          <Accordion>
+            <AccordionSummary expandIcon={<ExpandMoreTwoTone />}>
+              <Typography sx={{ fontSize: '18px' }} className="no-margin">
+                Data Access Features
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails sx={{ padding: 0 }}>
+              <SynapsePriceFeatureRow
+                featureDescription="Data access management"
+                standardPlanValue="Self-Service*"
+                managedPlanValue="Self-Service"
+                dataCoordinationPlanValue="Custom"
+              />
+              <SynapsePriceFeatureRow
+                featureDescription="Seamless data access integration with compute environments"
+                dataCoordinationPlanValue={true}
+              />
+              <SynapsePriceFeatureRow
+                featureDescription="Customized data portal"
+                standardPlanValue="Basic"
+                managedPlanValue="Basic"
+                dataCoordinationPlanValue={true}
+              />
+              <SynapsePriceFeatureRow
+                featureDescription="Contract length"
+                managedPlanValue="Length of grant"
+                dataCoordinationPlanValue="Contact Sage‡"
+              />
+            </AccordionDetails>
+          </Accordion>
+
+          <Typography
+            className="text-align-left"
+            sx={{
+              fontWeight: 300,
+              fontSize: '12px',
+              lineHeight: '16px',
+              paddingTop: '15px',
+            }}
+          >
+            * Sage can create a customized quote for you if, for example, you
+            need to store {'>'}100GB, get help with public data sharing, or want
+            a “click wrap” for displaying terms of data sharing. You may also
+            qualify for free storage depending on your program.
+            <br />
+            ** Due to the level of customization afforded, this option does not
+            follow a fixed pricing model.
+            <br />
+            *** Size limitations are removed and regions can be customized if
+            you use your own cloud storage bucket.
+            <br />‡ 5 years or the length of your grant if you contact us for a
+            custom quote.
+          </Typography>
         </div>
       </div>
 
-      <div className="SynapseHomepage__Section OffWhiteBackground">
+      {/* /**
+       * 
+Data access management
+
+Seamless data access integration with compute environments 
+
+Customized data portal 
+
+Contract length
```

Comment:
Is this future content that is coming later, or should be deleted?

Reply:
Haha, should be deleted.  (it's a scratch pad while adding content!)